components/ide/jetbrains/image/status/main.go

```diff
@@ -89,10 +90,27 @@ func main() {
 	log.Info(ServiceName + ": " + Version)
 	startTime := time.Now()
 
-	if len(os.Args) < 3 {
-		log.Fatalf("Usage: %s <port> <kind> [<link label>]\n", os.Args[0])
+	var port string
+	var warmup bool
+
+	if len(os.Args) < 2 {
+		log.Fatalf("Usage: %s (warmup|<port>)\n", os.Args[0])
+	}
+
+	if os.Args[1] == "warmup" {
+		if len(os.Args) < 2 {
+			log.Fatalf("Usage: %s <alias>\n", os.Args[0])
```

Comment:
Just noticed here it should be something like this instead:
```suggestion
		if len(os.Args) < 3 {
			log.Fatalf("Usage: %s %s <alias>\n", os.Args[0], os.Args[1])
```
Because `os.Args[0]` is `status`, and `os.Args[1]` is `warmup`.

Reply:
good call, I was actually making this change, will use your code, thanks